DESCRIPTION

```diff
@@ -68,6 +68,7 @@ Config/renv/profiles/reporting/dependencies:
     leaflet,
     magrittr,
     markdown,
+    noctua,
```

Comment:
**issue (blocking):** We can't have the pipeline/reports doing Athena queries during a run. If there's data you need but don't have, work with Billy to add it to the data that gets pulled in the `ingest` stage of the pipeline.